### ISSUE TITLE
The game fail when I try to select [Preset Character]

### ISSUE THREAD

**Issue by mdtrooper** [CleverRaven/Cataclysm-DDA#13703]
This is my system:
```
$ uname -a
Linux speccy-laptop 3.12.0-031200-generic #201311031935 SMP Mon Nov 4 00:36:54 UTC 2013 x86_64 x86_64 x86_64 GNU/Linux

$ lsb_release -a
No LSB modules are available.
Distributor ID:	LinuxMint
Description:	Linux Mint 17.2 Rafaela
Release:	17.2
Codename:	rafaela
```

The game version is: 0.C-6261-gb47752d

This is the output in the shell:

```
$ ./cataclysm-tiles
/usr/include/c++/4.8/debug/vector:346:error: attempt to subscript container 
    with out-of-bounds index 0, but container only holds 0 elements.

Objects involved in the operation:
sequence "this" @ 0x0x64565b8 {
  type = NSt7__debug6vectorISsSaISsEEE;
}
Aborted (core dumped)
```

The core dumped file is named as .png (thanks github for nothing).

![core 7z](https://cloud.githubusercontent.com/assets/1285451/10238919/1d3b34da-68c6-11e5-9f5a-ef53f6c5d481.png)

**Comment by mdtrooper** [comment]
And the file is 7ziped.

**Comment by mdtrooper** [comment]
Don't worry, the bug is about the game has not a created world (I don't know how it was deleted).

Maybe the solution is check if there is any world before to load preset character.

**Comment by ejseto** [comment]
Not only is this the 5th time this has been reported, but it is already fixed. Update your build.

**Comment by narc0tiq** [comment]
Probably fixed by #13668, about three days ago now (your build of b47752d is approximately 12 days old). Please reopen if that appears to not be the case.
